JobConfig/digitize/Extracted.fcl

```diff
@@ -29,4 +29,6 @@ physics.producers.CrvCoincidence.PEthresholds : [  8 ,  8 ,  8 ]
 physics.producers.CrvCoincidence.adjacentPulseTimeDifferences  : [ 10 , 10 , 10 ]
 physics.producers.CrvCoincidence.maxTimeDifferences            : [ 10 , 10 , 10 ]
 physics.producers.CrvCoincidence.coincidenceLayers             : [  3 ,  3 ,  3 ]
```

Comment:
CrvCoincidence is an outdated module. It is superseded by CrvCoincidenceClusterFinder.
Shouldn't the settings for CrvPhotons and CrvCoincidenceClusterFinder come from Offline/CRVResponse/fcl/epilog_extracted.fcl?